doc/glossary.rst

```diff
@@ -1598,11 +1599,11 @@ functions or non-estimator constructors.
     ``warm_start``
 
         When fitting an estimator repeatedly on the same dataset, but for
-        multiple parameter values (such as to find the value maximizing
+        multiple parameter values (such as to find the value-maximizing
```

Comment:
please revert, the previous one was correct

(what needs to be found is the value, not the performance)

Reply:
If you find it unclear you can replace by e.g.

"the value that maximizes performance"